x/genutil/types/genesis_state.go

```diff
@@ -20,6 +20,10 @@ type GenesisState struct {
 
 // NewGenesisState creates a new GenesisState object
 func NewGenesisState(genTxs []json.RawMessage) GenesisState {
```

Comment:
I don't think this is the correct approach, although it may not hurt, but I don't think we should do this unless we do it for all modules.

See: https://github.com/cosmos/cosmos-sdk/pull/5499/files#diff-617fab2f71b664452fdd126c60ca4aaf

Reply:
I'd be happy to close that PR in favor of yours with the changes upstreamed 👍